rewrite-yaml/src/main/java/org/openrewrite/yaml/CoalesceProperties.java

```diff
@@ -16,20 +16,29 @@
 package org.openrewrite.yaml;
 
 import org.openrewrite.Recipe;
+import org.openrewrite.yaml.search.FindIndentYamlProcessor;
 import org.openrewrite.yaml.tree.Yaml;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class CoalesceProperties extends Recipe {
     public CoalesceProperties() {
-        this.processor = () -> new CoalescePropertiesProcessor<>();
+        this.processor = () -> new org.openrewrite.yaml.CoalesceProperties.CoalescePropertiesProcessor<>();

```

Comment:
Not a critique, just for learning purposes-- are there specific reasons to use the fully qualified name here? Assuming it's for style/clarity in nested-class cases like this, though I'm curious if there's more to it.

Reply:
ooh, that is not a change I meant to make. I probably need to reset my intellij settings.